app.py

```diff
@@ -58,12 +63,13 @@ def get_todos():
 
 @app.route("/todos", methods=["POST"])
 @requires_auth
-@aserto.authorize
+@aserto.check(policyRoot="rebac", objType="resource-creator", objId="resource-creators", relationName="member").authorize
```

Comment:
This line results in a SyntaxError when running in Python 3.8.
It works fine on 3.9+ but feels a bit clunky. Is it really necessary to require the `.authorize` at the end?

Reply:
yes, the `.check` method on the AsertoMiddleware instantiates a CheckMiddleware that has the `.authorize` call on it